Caddyfile

```diff
@@ -3,7 +3,7 @@
 	admin off
 }
 
-:80 {
+:9080 {
```

Comment:
Can you set this as an ENV variable that defaults to 80? That way we can avoid the breaking change but still allow others to configure as necessary. Once we move to v1 I can make the port change by default.

https://caddyserver.com/docs/caddyfile-tutorial#environment-variables

Reply:
good idea, will do.